documentation/proc-pages/development/standards.md

```diff
@@ -94,32 +132,102 @@ ii
 
 | Variable name | Description | Units |
 | ------------- | ----------- | :---: |
-| `i_plasma`    | Plasma current | A |
-| `i_plasma_ma` | Plasma current | MA |
+| `plasma_current`    | Plasma current | A |
+| `plasma_current_MA` | Plasma current | MA |
 | `b_t_onaxis`  | Toroidal field on-axis | T |
 | `b_t_max`     | Max toroidal field | T |
 | `n_electron_vol` | Volume average electron density | m-3 |
 | `t_electron_vol_ev` | Volume avgerage electron temperature | eV |
-| `m_steel` | Mass of steel | kg |
-| `m_steel_tonne` | Mass of steel | tonne |
+| `mass_steel` | Mass of steel | kg |
+| `mass_steel_tonne` | Mass of steel | tonne |
 | `e_neutron_ev` | Energy of neutron | eV |
-| `e_neutron_mev` | Energy of neutron | MeV |
+| `e_neutron_MeV` | Energy of neutron | MeV |
 | `v_tf_dump` | TF dump voltage | V |
 | `time_plant_life` | Plant lifetime | s |
 | `time_plant_life_yrs` | Plant lifetime | years |
 | `dr_tf_inboard_leg` | TF coil inboard leg radial thickness | m |
 | `dr_blanket_inboard` | Inboard blanket thickness | m |
 | `velocity_coolant` | TF centrepost coolant velocity | m/s |
-| `vol_plasma` | Plasma volume | m3 |
-| `a_plasma` | Plasma area | m2 |
+| `plasma_volume` | Plasma volume | m3 |
+| `plasma_area` | Plasma area | m2 |
 | `angle_div_target` | Divertor target angle | radians |
 | `angle_div_target_deg` | Divertor target angle | deg |
 | `sig_tf_r` | TF radial stress  | Pa |
 | `` |  |  |
 
-Please see issue 940 <a href = "https://github.com/ukaea/PROCESS/issues/940"> to discuss new conventions.
+Please see issue [#940](https://github.com/ukaea/PROCESS/issues/940) to discuss new conventions.
+
+## Type-Hints
+
+It is greatly encouraged and recommended to include type hints for all inputs and outputs in Python. Please follow the guidelines set out in [PEP-484](https://peps.python.org/pep-0484/).
+
+## Docstrings
+
+### Functions
+
+If writing in new Python functions please use the docstring template below.
+
+```python
+def function_name(param1, param2):
+    """
+    Brief description of what the function does.
+
+    Args:
+        param1 (type): Description of the first parameter with units.
+        param2 (type): Description of the second parameter with units.
+
+    Returns:
+        return_type: Description of the return value with units.
+
+    Notes:
+        Detailed description of the function. This can include information about the algorithm,
+        any important notes, and other relevant details.
+
+    References:
+        - Reference 1 
+        - Reference 2
+         (Use a verbose reference style that has a clickable DOI or HTML link.)
+    """    
```

Comment:
I believe this is a modified version of the numpy (or is it Google?) docstring format; currently we use the sphinx/reStructuredText one. I'd be keen to use one format, and then convert using a tool (pretty sure I've found ones before) if we want a different one. I don't particularly like the sphinx one, but it's widely used. Perhaps make it explicit which format we follow here, and reference the autodocstring (or equivalent) extensions in VSCode.

Reply:
It is a variant of the Google docstring style, though with some additions as we cite papers etc. I will add this note and also a possible extension